docs/source/run_model_in_simulation.rst

```diff
@@ -84,17 +116,25 @@ Now we are ready to launch the simulation::
       --cacheline_size=32 \
       --accel_cfg_file=gem5.cfg \
       --fast-forward=10000000000 \
-      -c <path-to-smaug>/build/bin/smaug \
+      -c /workspace/smaug/build/bin/smaug \
       -o "my_model_topo.pbtxt my_model_params.pb --gem5 --debug-level=0"
 
-This command runs our custom 3-level model in gem5-Aladdin simulation.
+This command runs our custom 3-level DNN model in gem5-Aladdin simulation.
 gem5-Aladdin provides a wide range of SoC simulation choices, for instance,
 here, the simulated SoC has an out-of-order CPU running at 2.5GHZ, a two-level
 cache hierarchy with a 2MB, 16-way associative L2 cache and 32B cacheline size.
-The :code:`fast-forward` parameters is used to speed up the simulation of
-the initialization phase, which uses a simplified CPU model in gem5. In SMAUG,
-we use gem5's magic instruction :code:`m5_switch_cpu` to switch to the detailed
-OoO CPU when the initialization is done.
+
+There are a few parameters that have special importance. :code:`--mem-size=4GB`
```

Comment:
Turning this paragraph into a list would make it more easily readable/skimmable.

Reply:
Right. Done.